package.json

```diff
@@ -6,13 +6,6 @@
   "module": "./ts/dist/esm/node.js",
   "types": "./ts/dist/types/index.d.ts",
   "browser": "./ts/dist/esm/index.js",
-  "exports": {
```

Comment:
I also removed the `exports` field as I cannot find a combination of directives that works properly for Node.js, Jest, and Webpack environments at the same time. It's a new and optional field and everything seems to do the right thing once this is removed.